### ISSUE TITLE
Luv and LCHuv gives incorrect color values for low lightness cyan

### ISSUE THREAD

**Issue by jensln** [JuliaGraphics/Colors.jl#534]
The Luv and LCHuv color spaces jumps to an unexpected hue even though the color parameters are constant and only the lightness is varying. These images speaks for themselves. Minimal reproducible example in Pluto, showing the incorrect colors:

![image](https://github.com/JuliaGraphics/Colors.jl/assets/71942129/45b700d8-4318-4abe-89b1-b78e57edd7b9)

Also visible [in the website](https://juliagraphics.github.io/Colors.jl/stable/constructionandconversion/), choosing the relevant parameters, as demonstrated on my phone:

<img src="https://github.com/JuliaGraphics/Colors.jl/assets/71942129/40778508-44fb-4658-8ccb-db63f15c7b0a" width="300" />

<img src="https://github.com/JuliaGraphics/Colors.jl/assets/71942129/338f5b42-4b47-4f08-b188-da45f506be1e" width="300" />

**Comment by kimikage** [comment]
Basically, Colors.jl does not perform gamut clamping in color conversion.

Since `RGB{N0f8}` can only represent the component values between `0` and `1`, it is clamped by the component type restriction, but that does not imply "clamping by gamut”.
In fact, `RGB{Float64}(1.2, 0.0, -0.2)` is a valid value in a sense （whether or not it is handled well).

Even if a color cannot be represented in RGB (especially sRGB), that color may be represented in other color models or color gamuts.
Therefore, on the part of `Luv`, the value cannot be restricted without consensus or standard rules.
In other words, this is a problem on the RGB side, the destination of the conversion.

However, as far as I know, there is no low-cost and accurate way to evaluate whether a given Luv color falls within the sRGB gamut.
This is the reason why such false colors are left unchecked.

To begin with, the cost of conversion between Luv and RGB is expensive.
The additional computing cost may not be a concern.
I do not know about the labor costs.
